.github/workflows/tests.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       - run: bash scripts/environment/prepare.sh
       - run: make slim-builds
       - run: make test-shutdown
-      # Currently has too few tests to justify a separate task in CI.
+      - run: make test-behavior
```

Comment:
It seems like this is also run by `test-behavior`. Do we need it here too?

Reply:
Whoops I grabbed the wrong one! Nice catch! omg!